Gemfile.lock

```diff
@@ -364,11 +363,12 @@ GEM
       rainbow (>= 2.2.2, < 4.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 1.7)
-    rubocop-rails (2.4.0)
+    rubocop-rails (2.5.0)
```

Comment:
### [rubocop-rails](https://docs.rubocop.org/projects/rails/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.4.0 → 2.5.0

[_[change-log](https://github.com/rubocop-hq/rubocop-rails/blob/master/CHANGELOG.md), [source-code](https://github.com/rubocop-hq/rubocop-rails/)_]





<details>
<summary>Commits</summary>

A change of **58** commits. See the full changes on [the compare page](https://github.com/rubocop-hq/rubocop-rails/compare/v2.4.0...v2.5.0).

These are the first 10 commits:
- (9db2fc9) [Fix a typo](https://github.com/rubocop-hq/rubocop-rails/commit/9db2fc97fc7f025f30d71cd7408216a50b0d38db)
- (b50c142) [Use squiggly heredoc](https://github.com/rubocop-hq/rubocop-rails/commit/b50c142b638245e3239dad9d85df2525b317f79d)
- (1ab92f2) [Merge pull request #160 from koic/use_squiggly_heredoc](https://github.com/rubocop-hq/rubocop-rails/commit/1ab92f2067dbf522a66b635648259b61e9fb81e1)
- (4d79291) [FIX #159 enum hash auto correct for % arrays](https://github.com/rubocop-hq/rubocop-rails/commit/4d79291f20d0dbf446a0389fbf7d8b2819345210)
- (5707bb9) [Merge pull request #163 from ngouy/FIX-enum-hash-auto-correc…](https://github.com/rubocop-hq/rubocop-rails/commit/5707bb91e2581ce91e3fcaa92fbc362e6912acad)
- (e43c772) [Copyedit RakeEnvironment cop documentation](https://github.com/rubocop-hq/rubocop-rails/commit/e43c7729d7de6d1f9d33caf73f68e1e60fbaf843)
- (8d48d8d) [Merge pull request #164 from dylnclrk/edit-rake-environment-…](https://github.com/rubocop-hq/rubocop-rails/commit/8d48d8dd70ec42fb7c3f518db1698a712288086e)
- (36d736b) [Workaround for broke CI with YARD 0.9.20](https://github.com/rubocop-hq/rubocop-rails/commit/36d736bfc81cdac6b4190ffaf69f6691827f9522)
- (cd182c0) [Don't register BulkChangeTable offense if block intervenes](https://github.com/rubocop-hq/rubocop-rails/commit/cd182c06065d012999d1eaf7f12468038ad35848)
- (1fc2d47) [Merge pull request #169 from koic/workaround_for_yard_0_9_20](https://github.com/rubocop-hq/rubocop-rails/commit/1fc2d471d20c4bcfc8c774c4cd0a1cc03234b439)


</details>